sfcasts/ep1/computed.md

```diff
@@ -1,228 +1,192 @@
 
-The key thing here, if you inspect the element on that button, is that with `v-if`
-the contents are removed entirely. So if we click on the button again, we can see the
-contents shown, and if we hide it, they are gone! They are effectively added and
-removed from the DOM as the condition in the directive evaluates to `true` or `false`.
-That is *great*!
-
-The other way to hide things is by using `v-show`. With `v-show` things are going to
-look the same, but the difference is that when we click in `>> collapse` the content
-is always there. `v-show` just adds a style rule to the top element to set `display`
-to `none` when it needs to. *Sneaky*!
- 
-So `v-show` and `v-if` really accomplish the same task, they just do it in
-fundamentally different ways. So which one is more appropriate? It just depends on the
-situation. `v-show` is better when you're hiding and showing things a lot. When hiding
-and showing is really fast. If we have something that really won't be changing a lot,
-`v-if` is the way to go, because, if you use `v-if` here, for example, you won't be
-spending any time even rendering the sidebar until you actually need it. This is
-especially useful if you have several different layers of components here, where it
-can make your whole application load faster instead of loading all of these things
-just to hide them next.
+The key thing here, if you inspect the HTML, is that with `v-if`
+the contents are removed *entirely*. So if we click the button again, the contents
+come back, and if we collapse, they're gone!
 
-## .component style
+The other way to hide things is by using `v-show`. With `v-show` things will
+*look* the same, but the difference is that, when we click `>> collapse`, the HTML
+is still there. `v-show` just adds `display: none` to the element when it needs to.
+*Sneaky*!
 
-All right! While we're here, I also want to clean things up. We should refactor our
-`width` away from `style` and do this properly in a class. But before we do that,
-there is one thing I wanted to do earlier that I forgot about. You notice that down
-here in our styles, we have a single class called `sidebar`, and we're actually
-applying that to this element here in our template. This is great! But sometimes,
-as a convention, when you have a class that is applied to the root element in a
-component, you'll use the generic name `.component` 
+So `v-show` and `v-if` accomplish the same thing.
+Which you should use just depends on the
+situation. `v-show` is better if you're hiding and showing a lot - it's *super*
+fast. But if what you're hiding isn't shown often, like a modal or a different
+screen, `v-if` is better because Vue won't
+spend any time even rendering the element until you need it.
 
+## .component style
 
-as basically a way to say this is the main `class` in the template.
-Down here, update the class name to `.component`.
+All right! While we're here, I also want to clean things up. Let's refactor our
```

Comment:
The diff starts to fall apart here - so the rest of the PR is probably harder to read - it shows many more changes than there are.